docs/agent.rst

```diff
@@ -52,6 +48,21 @@ For TCP communication: ::
 
     $ ./MicroXRCEAgent tcp <port> [--discovery [<discovery_port>] ]
 
+For Serial communication (only Linux): ::
+
+    $ ./MicroXRCEAgent serial <device> [<baudrate>]
+
+The ``<device>`` is the device file that represents the serial port, and the ``<baudrate>`` is the baud rate of the communication which can take the following values:
+0, 50, 75, 110, 134, 150, 200, 300, 600, 1200, 1800, 240, 4800, 9600, 19200, 38400, 57600, 115200 (default), 230400, 460800, 500000, 576000, 921600, 1000000, 1152000, 1500000, 2000000, 2500000, 3000000, 3500000 or 4000000 Bd;
+any other value will be converted to ``int`` using the ``atoi`` function.
```

Comment:
Maybe `atoi` is a too tecnical *word* or implementation related. Is it add really information to the user? The same for `int`?

Reply:
I think the user must know how his/her input is processed when it is not a common baud rate.
It could be useful when the user wants to use a particular baud rate that is not in the baudrate table.